### PR TITLE
script for generating youtube crawler evaluation data

### PR DESCRIPTION
**Pull request by ljj7975**
The first milestone is generate a evaluation framework and dataset which we can measure our improvements.

the dataset need to be labelled, i.e. for given audio block. we have to check if the audio actually contains keyword.
`evaluation_data_generator.py` is the script which plays section of audio and ask how many times the keyword was spoken.
It will generate csv, where each column represent `url, start_ms, end_ms, cc_count, audio_count` where cc_count is how many it appeared in subtitle and audio_count is the user's response
This csv file will be what we will use for running evaluation

We may need to work on the same set of videos but putting video dataset on github is nonsense. 
`url_file_generator.py` generates .txt file containing valid urls.
Note that this script will be run once and .txt file generated will be used for `evaluation_data_generator.py`

### CODE REVIEW

**Comment by lemoz** [comment]
Amazing! Was going to create this but did not think to look at the captions to reduce the search space!

File: keyword_spotting_data_generator/evaluation/evaluation_data_generator.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
+import argparse
+import re
+import string
+import time
+import sounddevice as sd
+from pytube import YouTube as PyTube
```
Comment (daemon) [inline keyword_spotting_data_generator/evaluation/evaluation_data_generator.py:7]: Import blocks need one line between them, e.g.,
```
import argparse
import time

import requests
import six

from . import Foobar
```

File: keyword_spotting_data_generator/evaluation/evaluation_data_generator.py [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+import color_print as cp
+import utils as utils
+from youtube_searcher import YoutubeSearcher
+from youtube_crawler import YoutubeCrawler
+from url_file_reader import FileReader
+from evaluation_data_csv_writer import CsvWriter
+
+API_KEY = "< API_KEY >"
```
Comment (daemon) [inline keyword_spotting_data_generator/evaluation/evaluation_data_generator.py]: Maybe allow API key to be set using the commandline argument `--api_key`?

File: keyword_spotting_data_generator/evaluation/url_file_generator.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
+import argparse
+from pytube import YouTube as PyTube
+import color_print as cp
+from youtube_searcher import YoutubeSearcher
+import utils as utils
+
+API_KEY = "< API_KEY >"
```
Comment (daemon) [inline keyword_spotting_data_generator/evaluation/url_file_generator.py]: Here as well.

**Review by daemon — CHANGES_REQUESTED** [review]
Just minor issues.

**Pull request merged**